rabbitmq/Makefile

```diff
@@ -0,0 +1,45 @@
+RABBIT_VERSION  =3.8.0-management
+DOCKER_REGISTRY ?=itisfoundation
+
+VCS_URL          := $(shell git config --get remote.origin.url)
+VCS_REF          := $(shell git rev-parse --short HEAD)
+BUILD_DATE       := $(shell date -u +"%Y-%m-%dT%H:%M:%SZ")
+
+.PHONY: help
+help: ## displays targets
+ifeq ($(IS_WIN),)
+	@awk 'BEGIN {FS = ":.*?## "} /^[a-zA-Z_-]+:.*?## / {printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}' $(MAKEFILE_LIST)
+else
+	@awk 'BEGIN {FS = ":.*?## "} /^[a-zA-Z_-]+:.*?## / {printf "%-20s %s\n", $$1, $$2}' $(MAKEFILE_LIST)
+endif
+
+.DEFAULT_GOAL := help
+
+.PHONY: build
+build: ## builds image
+	@docker build --build-arg VERSION=$(RABBIT_VERSION) \
+		--label org.label-schema.schema-version=1.0 \
+        --label org.label-schema.build-date=${BUILD_DATE} \
+        --label org.label-schema.vcs-url=${VCS_URL} \
+        --label org.label-schema.vcs-ref=${VCS_REF} \
+		--tag $(DOCKER_REGISTRY)/rabbitmq:$(RABBIT_VERSION) \
+		.
+
+.PHONY: push
+push: ## pushes image to Dockerhub
+	@docker push $(DOCKER_REGISTRY)/rabbitmq:$(RABBIT_VERSION)
```

Comment:
should we add a check to ensure that only builds from ITISFoundation/Dockerfile and master are pushed??  This way the vsc info as commit has no ambiguities.
It already happened to us that we pushed to ITISFOundation from a fork repo with an older version even if the tag said that it was the latest

Reply:
I think in that case this is not an issue as this is a very simple image, basically only a configuration change.
One thing would be to add the branch name in the version of the image? this way it would prevent from pushing to the main image. But again I would favor this for more complex images, not this case.